microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -48,6 +48,11 @@ Updates for both single-version minor releases and patch releases are supported.
 //==== Control ingress for your use case with additional parameters
 //add phase 2 ingress control parameters announcement here
 
+[id="microshift-4-19-tls-config_{context}"]
+==== {microshift-short} control plane enhanced with TLS
+With this release, configurable transport layer protocols (TLS) on internal control plane components are enabled to help prevent known insecure protocols, ciphers, or algorithms to access the applications you run on {microshift-short}. You can choose to use either the TLS 1.2 or TLS 1.3 security profiles with {microshift-short}. For more information, see
```

Comment:
```suggestion
With this release, configurable transport layer protocols (TLS) on internal control plane components are enabled to prevent known insecure protocols, ciphers, or algorithms. You can use either the TLS 1.2 or TLS 1.3 security profiles to access the applications you run on {microshift-short}. For more information, see
```